models/sipnet/R/write.restart.SIPNET.R

```diff
@@ -34,6 +34,29 @@ write.restart.SIPNET<- function(out.dir, runid, time, settings, analysis.vec,
   
   settings$run$start.date <- paste0(time + 1,"/01/01") #TO FIX LATER #DONT WANT TO ASSUME YEARLY TIMESTEP
   settings$run$end.date <- paste0(time + 1,"/12/31")
+  
+  ## Converting to sipnet units
+  if(sample_parameters == TRUE){
+    load(file.path(settings$outdir, "samples.Rdata"))
+    prior.sla <- mean(ensemble.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA) #HACK
+  }else{
+    load(file.path(settings$outdir, paste0("ensemble.samples.",settings$state.data.assimilation$prior,".Rdata")))
+    prior.sla <- ens.samples[[which(names(ensemble.samples)!='soil')[1]]]$SLA
+  }
```

Comment:
I thought I'd mentioned this before, but if we need parameters values within write.restart.[MODEL], those parameters should be passed into the function as an argument rather than read from a file.